cscs-checks/apps/espresso/espresso_check.py

```diff
@@ -15,9 +15,15 @@ def __init__(self, variant, **kwargs):
                                        'Espresso')
         self.executable = 'pw.x'
         self.executable_opts = '-in ausurf.in'.split()
-
-        self.sanity_patterns = sn.assert_found(
-            r'convergence has been achieved', self.stdout)
+        property_patterns = {
```

Comment:
I don't see the reason for defining this dictionary here. You could simply do the following:

```python
energy = sn.extractsingle(...)
```

And use `energy` directly in the `sanity_patterns`.